workers/unity/Assets/Generated/Source/improbable/gdk/tests/ExhaustiveBlittableSingularGameObjectComponentDispatcher.cs

```diff
@@ -44,6 +44,7 @@ internal class GameObjectComponentDispatcher : GameObjectComponentDispatcherBase
 
             private const uint componentId = 197720;
             private static readonly InjectableId readerWriterInjectableId = new InjectableId(InjectableType.ReaderWriter, componentId);
+            private static readonly InjectableId commandRequestHandlerInjectableId = new InjectableId(InjectableType.CommandRequestHandler, componentId);
```

Comment:
should command request injectable id still be created if there is nothing that will be injected?

Reply:
Added an if condition to only generate this line if the component has commands.